hilti/toolchain/src/config.cc.in

```diff
@@ -67,9 +88,15 @@ void Configuration::init(bool use_build_directory) {
     runtime_cxx_flags_debug = flatten({prefix("${HILTI_CONFIG_RUNTIME_INCLUDE_DIRS}", "-I", installation_tag),
                                        prefix("${HILTI_CONFIG_RUNTIME_CXX_FLAGS_DEBUG}", "", installation_tag)});
 
+    if ( auto libhilti_pch = precompiled_libhilti(*this, true) )
+        runtime_cxx_flags_debug.push_back(rt::fmt("-include%s", libhilti_pch->c_str()));
```

Comment:
Forgot to ask about this last time: is it on purpose that there's no space after `-include`?

Reply:
Yes, this is on purpose. I found some clang versions had issues with spaces there.